apps/beets-frontend-v3/package.json

```diff
@@ -52,7 +52,7 @@
     "@repo/typescript-config": "workspace:*",
     "@types/js-cookie": "^3.0.6",
     "@types/lodash": "^4.17.17",
-    "@types/node": "22.15.19",
```

Comment:
<img width="903" alt="Screenshot 2025-06-24 at 08 51 44" src="https://github.com/user-attachments/assets/bf07667e-3047-4d8c-aa5c-031c799645b4" />

It seems that is not a Vercel supported version, we should continue with 22.x.x.

Reply:
ah i didn't check vercel support, reverted that commit